includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1770,7 +1771,7 @@ public function update_option( $key, $value = '' ) {
 	public function update_is_woopay_enabled( $is_woopay_enabled ) {
 		$current_is_woopay_enabled = 'yes' === $this->get_option( 'platform_checkout', 'no' );
 		if ( $is_woopay_enabled !== $current_is_woopay_enabled ) {
-			wc_admin_record_tracks_event(
+			WC_Payments::woopay_tracker()->maybe_record_admin_event(
 				$is_woopay_enabled ? 'woopay_enabled' : 'woopay_disabled',
```

Comment:
Just confirming that the events are now `wcadmin_woopay_enabled` and `wcadmin_woopay_disabled` and not `platform_checkout_enabled` and `platform_checkout_disabled`

Reply:
Yes. This is correct. These events are renamed as a part of recently merged naming convention updates https://github.com/Automattic/woocommerce-payments/pull/6061